src/io/github/TheBusyBiscuit/GitHubWebAPI4Java/objects/repositories/GitHubRepository.java

```diff
@@ -1129,4 +1130,33 @@ public GitHubLicense getLicense() throws IllegalAccessException, UnsupportedEnco
 		
 		return isInvalid(response, "license") ? null: new GitHubLicense(api, response.get("license").getAsJsonObject().get("key").getAsString(), response.get("license").getAsJsonObject());
 	}
+
+    /**
+     * Returns a list of thirty of the latest releases of this repository.
+     * The GitHub API does not return all the releases available on a repository, and the number of releases returned by this call might change in the future.
+     * @return a list of the latest releases of this repository.
+     * @throws IllegalAccessException
+     * @since 1.3.4
+     */
+	@GitHubAccessPoint(path = "/releases", type = GitHubRelease.class, requiresAccessToken = false)
+	public List<GitHubRelease> getReleases() throws IllegalAccessException {
+        GitHubObject releases = new GitHubObject(api, this, "/releases");
+        JsonElement response = releases.getResponse(true);
+
+        if (response == null) {
+            throw new IllegalAccessException("Could not connect to '" + getURL() + "'");
+        }
+
+        List<GitHubRelease> list = new LinkedList<>();
+        JsonArray array = response.getAsJsonArray();
+
+        for (int i = 0; i < array.size(); i++) {
+            JsonObject object = array.get(i).getAsJsonObject();
+
+            GitHubRelease release = new GitHubRelease(api, this, object.get("id").getAsInt(), object);
+            list.add(release);
+        }
+
+        return list;
+    }
```

Comment:
What about pagination? I am sure there can be multiple pages with this one.

Reply:
I can't double-check right now because I've just been rate-limited (bruh), but I don't think there's any sort of pagination. It returns 30 releases, and that's about it, as far as I know.

Reply:
Just checked, there are indeed pages:
https://api.github.com/repos/BentoBoxWorld/BentoBox/releases?page=1
https://api.github.com/repos/BentoBoxWorld/BentoBox/releases?page=2

Reply:
The Tags API is also paged and the #getTags() method is implemented similarly to #getReleases(), so I guess there's no additional stuff to add.